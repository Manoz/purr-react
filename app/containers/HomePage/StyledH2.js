import styled from 'styled-components';
import H2 from 'components/H2';

const StyledH2 = styled(H2)`
  padding-bottom: 1rem;
  border-bottom: 1px solid #b7b7b7;
`;

export default StyledH2;
