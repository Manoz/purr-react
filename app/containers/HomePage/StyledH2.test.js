import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import StyledH2 from './StyledH2';

describe('<StyledH2 />', () => {
  it('should match the snapshot', () => {
    const wrapper = renderer
      .create(<StyledH2>test</StyledH2>)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });

  it('should adopt a valid attribute', () => {
    const id = 'test';
    const wrapper = shallow(<StyledH2 id={id} />);
    expect(wrapper.prop('id')).toEqual(id);
  });
});
