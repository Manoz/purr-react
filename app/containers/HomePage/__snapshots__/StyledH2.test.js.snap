// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledH2 /> should match the snapshot 1`] = `
.c0 {
  margin-top: 1.2em;
  font-weight: 500;
  padding-bottom: 1rem;
  border-bottom: 1px solid #b7b7b7;
}

@media (min-width:768px) {
  .c0 {
    font-size: 24px;
    line-height: 1.20849;
    -webkit-letter-spacing: .015em;
    -moz-letter-spacing: .015em;
    -ms-letter-spacing: .015em;
    letter-spacing: .015em;
  }
}

@media (min-width:992px) {
  .c0 {
    font-size: 28px;
    line-height: 1.1073;
    -webkit-letter-spacing: .012em;
    -moz-letter-spacing: .012em;
    -ms-letter-spacing: .012em;
    letter-spacing: .012em;
  }
}

@media (min-width:1200px) {
  .c0 {
    font-size: 32px;
    line-height: 1.09375;
    -webkit-letter-spacing: .011em;
    -moz-letter-spacing: .011em;
    -ms-letter-spacing: .011em;
    letter-spacing: .011em;
  }
}

<h2
  className="H2-zasn0o-0 c0"
>
  test
</h2>
`;
